guix-science-nonfree/packages/nvidia-container-toolkit.scm

```diff
@@ -0,0 +1,203 @@
+;;; GNU Guix --- Functional package management for GNU
+;;; Copyright © 2024 Luciano Laratelli <luciano@laratel.li>
+;;; Copyright © 2024 Connor Clark <connor@psyleft.com>
+;;; Copyright © 2024 Ian FitzPatrick <ian@ianfitzpatrick.eu>
+;;;
+;;; This file is NOT part of GNU Guix, but is supposed to be used with GNU
+;;; Guix and thus has the same license.
+;;;
+;;; GNU Guix is free software; you can redistribute it and/or modify it
+;;; under the terms of the GNU General Public License as published by
+;;; the Free Software Foundation; either version 3 of the License, or (at
+;;; your option) any later version.
+;;;
+;;; GNU Guix is distributed in the hope that it will be useful, but
+;;; WITHOUT ANY WARRANTY; without even the implied warranty of
+;;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;;; GNU General Public License for more details.
+;;;
+;;; You should have received a copy of the GNU General Public License
+;;; along with GNU Guix.  If not, see <http://www.gnu.org/licenses/>.
+
+(define-module (guix-science-nonfree packages nvidia-docker)
+  #:use-module (guix gexp)
```

Comment:
Perhaps `nvidia-container` would be a better module/file name?

Reply:
I've changed the name to nvidia-container-toolkit as that's the name NVIDIA themselves use. I'd be fine with nvidia-container if you feel strongly about it ;-)

Reply:
I thought `nvidia-container` might work since `nvidia-container-toolkit` is only one of the packages in this module, but no big deal, I'm fine either way.